src/clients/twitch.ts

```diff
@@ -68,72 +93,26 @@ function registerTwitch(): void
-
-                    //[trailing bot message] This is a special method of recording the last twitch message because we need to have a method of removing this message after the fact
-                    const twitchMessage = new twitchMsg(msg, self, userState, channel);
-                    bridge.lastUserStateMsg = twitchMessage;
-
-                    //Adding an node intentionally without a discord message because this is the buffer message.
-                    const listNode = bridge.messageLinkdListInterface.addNode(new conjoinedMsg(undefined, [twitchMessage]));
-                    bridge.discordTwitchCacheMap.set(twitchMessage, listNode);
-                }
-
-                //before we override the last message, lets make sure we delete this twitch message if required (...this is jank)
-                //We wait until after the userstate message gets a bot message ID
-                if(bridge.lastUserStateMsg?.userState.cueForDelete)
-                {
-                    twitchDelete(bridge.lastUserStateMsg);
-                    manageMsgCache(bridge.discordTwitchCacheMap.get(bridge.lastUserStateMsg));
-                }
-
-                //If we found the twitch message we wanted to connect, we no longer need it in the cache
-                if(bridge.twitchMessageSearchCache[msg])
-                {
-
-                    const existingNode = bridge.twitchMessageSearchCache[msg],
-                        twitchMessage = new twitchMsg(msg, self, userState, channel);
-
-                    bridge.lastUserStateMsg = twitchMessage;
+            if(bridge.twitchMessageSearchCache[message])
+            {
+                const existingNode = bridge.twitchMessageSearchCache[message],
+                    twitchMessage = new twitchMsg(message, true, userState, channel);
 
-                    existingNode.data.twitchArray.push(twitchMessage);
-                    bridge.discordTwitchCacheMap.set(twitchMessage, existingNode);
+                existingNode.data.twitchArray.push(twitchMessage);
+                bridge.discordTwitchCacheMap.set(twitchMessage, existingNode);
 
-                    //Remove this from the cache since we found it
-                    delete bridge.twitchMessageSearchCache[msg];
-                }
+                //Remove this from the cache since we found it
+                delete bridge.twitchMessageSearchCache[message];
             }
         });
     }
 
     //Login to twitch
     //There are a lot of things that were named differently because this command has the potential of using environemnt variables, so this will be painful to look at and I'm sorry XP
-    if(configFile.T2D_DEV_SAVE_TOKEN && fs.existsSync('./devTwitchToken'))
-    {
-        //If the file exists, use the thing
-        console.log('==Found the token file! (devTwitchToken!) Using that instead of logging in via oauth (if this token is old, delete the file)==');
```

Comment:
I think this is useful to confirm if we're grabbing the token; otherwise we'd be worried that deploying over 50 times exhausts our token supply

I don't mind how it's phrased, weather it be this or another, but having some indication that we're using a previously saved token is pretty nice

Reply:
Impled.